CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Vx.x.x (Pre-release)
 
 ## Changes since v4.0.0
-- [#291][https://github.com/pusher/oauth2_proxy/pull/291] Added bash >= 4.0 dependency to configure script (@jmfrank63)
+- [#291](https://github.com/pusher/oauth2_proxy/pull/291) Added bash >= 4.0 dependency to configure script (@jmfrank63)
```

Comment:
I've also noticed this should probably be 292 as that was the PR
```suggestion
- [#292](https://github.com/pusher/oauth2_proxy/pull/292) Added bash >= 4.0 dependency to configure script (@jmfrank63)
```

Reply:
Fixed.